packages/manager/modules/pci/src/projects/project/kubernetes/service.js

```diff
@@ -205,6 +208,27 @@ export default class Kubernetes {
       );
   }
 
+  getRegions(projectId, ovhSubsidiary) {
+    const product = KUBE_PRODUCT_ID;
+    return this.$http
+      .get(`/cloud/project/${projectId}/capabilities/productAvailability`, {
+        params: { product, ovhSubsidiary },
+      })
+      .then(({ data }) =>
+        data.products
+          .find(({ name }) => name === product)
```

Comment:
Are we sure we will always find the product ?

Reply:
Yes, i got a confirmation from PCI.